pages/common/pixterm.md

```diff
@@ -0,0 +1,21 @@
+# pixterm
+
+> Image printing in the terminal.
+> See also: `chafa`, `catimg`.
+> More information: <https://github.com/eliukblau/pixterm>.
+
+- Print a static image to the terminal:
```

Comment:
```suggestion
- Print a static image:
```

Reply:
Similar suggestion by another reviewer on my `chafa` PR. I think it's important to mention that it's outputting to the terminal specifically, at least in the first example. Would this be a better clarification?

```suggestion
- Render a static image directly in the terminal:
```

Reply:
👍